src/main/java/com/flightstats/hub/webhook/WebhookManager.java

```diff
@@ -95,41 +95,78 @@ private void manageWebhook(Webhook daoWebhook, boolean webhookChanged) {
             logger.debug("found existing v2 webhook {}", name);
             Collection<String> servers = activeWebhooks.getServers(name);
             if (servers.isEmpty()) {
-                callOneRun(name, hubCluster.getRandomServers());
+                callOneRun(name, getOrderedServers());
             } else if (webhookChanged) {
                 callOneRun(name, servers);
             }
         } else {
             logger.debug("found new v2 webhook {}", name);
-            callOneRun(name, hubCluster.getRandomServers());
+            callOneRun(name, getOrderedServers());
         }
     }
 
+    /**
```

Comment:
If the balancing doesn't end up giving the amount of improvement we want, I'm wondering if some other metric that corresponds to webhook load would be a good alternative.